Can't parse json object traversal

Given a bicep line like:
var subscriptions = json(loadTextContent('./vars.json')).subscriptions

and vars.json contents like:
```
{
  "subscriptions": {
    "prod": {
      "id": "an id"
    }
  }
}
```
pycep will fail on "[...] .subscriptions"
This syntax is syntactically correct and briefly referenced in [official documentation](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/bicep-functions-files#remarks-1)

```
2022-05-15 22:02:40,670 [MainThread  ] [ERROR]  [bicep] Couldn't parse test_json.bicep
Traceback (most recent call last):
  File "/home/username/.local/lib/python3.10/site-packages/lark/lexer.py", line 528, in lex
    yield lexer.next_token(lexer_state, parser_state)
  File "/home/username/.local/lib/python3.10/site-packages/lark/lexer.py", line 466, in next_token
    raise UnexpectedCharacters(lex_state.text, line_ctr.char_pos, line_ctr.line, line_ctr.column,
lark.exceptions.UnexpectedCharacters: <unprintable UnexpectedCharacters object>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/username/.local/lib/python3.10/site-packages/checkov/bicep/parser.py", line 20, in parse
    template = self.bicep_parser.parse(text=content)
  File "/home/username/.local/lib/python3.10/site-packages/pycep/main.py", line 23, in parse
    tree = self._create_tree(text=text, file_path=file_path)
  File "/home/username/.local/lib/python3.10/site-packages/pycep/main.py", line 41, in _create_tree
    return self.lark_parser.parse(bicep_text)
  File "/home/username/.local/lib/python3.10/site-packages/lark/lark.py", line 625, in parse
    return self.parser.parse(text, start=start, on_error=on_error)
  File "/home/username/.local/lib/python3.10/site-packages/lark/parser_frontends.py", line 96, in parse
    return self.parser.parse(stream, chosen_start, **kw)
  File "/home/username/.local/lib/python3.10/site-packages/lark/parsers/lalr_parser.py", line 41, in parse
    return self.parser.parse(lexer, start)
  File "/home/username/.local/lib/python3.10/site-packages/lark/parsers/lalr_parser.py", line 171, in parse
    return self.parse_from_state(parser_state)
  File "/home/username/.local/lib/python3.10/site-packages/lark/parsers/lalr_parser.py", line 188, in parse_from_state 
    raise e
  File "/home/username/.local/lib/python3.10/site-packages/lark/parsers/lalr_parser.py", line 178, in parse_from_state 
    for token in state.lexer.lex(state):
  File "/home/username/.local/lib/python3.10/site-packages/lark/lexer.py", line 537, in lex
    raise UnexpectedToken(token, e.allowed, state=parser_state, token_history=[last_token], terminals_by_name=self.root_lexer.terminals_by_name)
lark.exceptions.UnexpectedToken: <unprintable UnexpectedToken object>
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Support property accessor for `json` function
+
 ## [0.3.5] - 2022-05-15
 
 ### Added

--- a/pycep/bicep.lark
+++ b/pycep/bicep.lark
@@ -175,7 +175,7 @@ int_func: ("int" | "sys.int") "(" value_brackets ")"
 
 ?function_object: json_func
 
-json_func: ("json" | "sys.json") "(" value_brackets ")"
+json_func: ("json" | "sys.json") "(" value_brackets ")" ["." STRING]
 
 // functions - resource
 

--- a/pycep/transformer.py
+++ b/pycep/transformer.py
@@ -780,13 +780,20 @@ class BicepToJson(Transformer[Token, pycep_typing.BicepJson]):
     #
     ####################
 
-    def json_func(self, args: tuple[pycep_typing.PossibleValue]) -> pycep_typing.Json:
-        return {
+    def json_func(self, args: tuple[pycep_typing.PossibleValue, pycep_typing.PossibleNoneValue]) -> pycep_typing.Json:
+        arg_1, property_name = args
+
+        result: pycep_typing.Json = {
             "type": "json",
             "parameters": {
-                "arg_1": args[0],
+                "arg_1": arg_1,
             },
         }
+
+        if property_name:
+            result["property_name"] = str(property_name)
+
+        return result
 
     ####################
     #

--- a/pycep/typing.py
+++ b/pycep/typing.py
@@ -590,6 +590,7 @@ class _JsonParameters(TypedDict):
 class Json(TypedDict):
     type: Literal["json"]
     parameters: _JsonParameters
+    property_name: NotRequired[str]
 
 
 ####################

--- a/tests/test_function/examples/object/json/main.bicep
+++ b/tests/test_function/examples/object/json/main.bicep
@@ -19,7 +19,7 @@ resource identityProvider 'Microsoft.ApiManagement/service/identityProviders@202
 }
 
 // output
-output ftpUser string = json(site.properties)
+output ftpUser string = json(site.properties).id
 
 // namespace sys
 output namespace string = sys.json(site)

--- a/tests/test_function/examples/object/json/result.json
+++ b/tests/test_function/examples/object/json/result.json
@@ -65,7 +65,8 @@
           "type": "json",
           "parameters": {
             "arg_1": "site.properties"
-          }
+          },
+          "property_name": "id"
         }
       }
     },

